src/passthrough/backend.rs

```diff
@@ -50,10 +50,10 @@ impl Backend {
     ) -> Backend {
         let pool = ServerPool::new_servers(servers);
         Backend {
-            name: name,
+            name,
```

Comment:
if the variable has the same name as the field it is good practice to just send the variable.